examples/contracts/rock_paper_scissors.py

```
@@ -0,0 +1,78 @@
+Respond using ONLY the following format:
+{
+"move": str
+}
+It is mandatory that you respond only using the JSON format above,
+nothing else. Don't include any other words or characters,
+your output must be only JSON without any formatting prefix or suffix.
+This result should be perfectly parseable by a JSON parser without errors.
+"""
+
+        def nondet():
+            res = gl.nondet.exec_prompt(prompt)
```

Ruff (0.15.5)

[error] `gl` may be undefined, or defined from star imports

(F405)